src/core/Config.js

```diff
@@ -20,6 +20,11 @@ class Config {
 
       if (typeof configuration === "function") {
         configuration = await configuration();
+        configuration.defaultResponse = configuration.defaultResponse || {
+          name: "AXE API",
+          description: "The best API creation tool in the world.",
+          aim: "To kill them all!",
+        };
```

Comment:
I think this is wrong. Please review the full method, `load()`. It creates a loop for all configuration files that are located under the `app/Config` directory. So, you try to load `defaultResponse` property for all configuration files. 

Actually, we don't have to do anything in this file. If you put a `defaultResponse` property to the [Application Configuration](https://axe-api.github.io/config/#application), it would works.

Reply:
@YusufSina, do you need some help?

Reply:
Hi @YusufSina, 

I hope you doing well. But I am afraid that I am going to close this PR soon if you can not make a time. :cry: